### ISSUE TITLE
enable selection of unavailable size variant

### ISSUE THREAD

**Issue by briznad** [Land-of-Distraction/LoD.com#14]
**Is your improvement related to a problem? Please describe.**
Currently when a size is unavailable it is disabled/appears greyed out and cannot be selected. This means the customer cannot choose this size, even if that's the size they want, which would show the restock notification signup, allowing them to know when the size is again avail.

**Describe the solution you'd like**
Allow disabled sizes to be selected. This change should only affect the selecting behavior; the styling/display of disabled items should not change.

**Comment by briznad** [comment]
this is the follow on issue to #6

**Comment by briznad** [comment]
@ErinLOD can you pls assign priority?

**Comment by briznad** [comment]
This issue is unfixable currently. I went pretty far into changing the functionality to match the desired behavior, but ran into repeated unintended consequences. The final roadblock was that if you clicked on a size variant that wasn't available, the in stock notification screen would replace the Add to Cart button, as desired. But if you then selected an available size the Add to Cart button wouldn't return. I found that this was due to code not written by Rockwell, but coming from a Shopify component, meaning I don't have access to change it.

My conclusion is that the behavior we are attempting to implement runs counter to how Shopify assumes one would use the out of stock notification. Therefore it should only be used to notify on products where all variants are unavailable.

I've reverted all code back to it's original state.
